packages/stake/src/__tests__/stake.test.ts

```diff
@@ -0,0 +1,39 @@
+import { epoch } from '@vanilladefi/core-sdk'
+import { formatUnits } from 'ethers/lib/utils'
+import { getLeaderboard, getUserJuiceDelta } from '../stake'
+import { testPolygonProvider } from '../__utils__/utils'
+const hardhatTestAccount = '0x35434a35C19cabC4bD508Edf538cC1Cf33BDA688'
+
+test('Get user JUICE delta', async () => {
+  const delta = await getUserJuiceDelta(hardhatTestAccount, epoch, 'latest', {
```

Comment:
You could refactor all these options to a const to improve the test case readability.